avatars/avatars.js

```diff
@@ -993,7 +993,7 @@ class Avatar {
     this.lastNeedsHeadTarget = false;
     this.lastHeadTargetTime = -Infinity;
 
-    this.manuallySetMouth=false;
+    this.disableAudioWorkerSetVolume=false;
```

Comment:
The default seems to be `true` when the grunt ends... so why is this false. Probably a consequence of choosing a poor variable name.

Reply:
Got it. [set disableAudioWorkerSetVolume initial value to true](https://github.com/webaverse/app/pull/3651/commits/cb447d56a76c188fa794dded6a04371269565571)